lib/writeLog.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+var config = require('../config/secrets');
+var logger = require('./logger');
+
+var logFlag = config.get().logger.api.log_user_activity;
+
+var writer = {};
+
+writer.writeIncident = function(req, incident, action) {
+  if (logFlag === false) return {};
```

Comment:
What about `if (!logFlag) return;`? It doesn't look like the return value of `writeIncident` or any of its friends is ever used.